_posts/2021-12-16-ngff.md

```diff
@@ -0,0 +1,142 @@
+with large bioimaging datasets-- public data resources, high content screening
+and light sheet microscopy. We have tried to make the work as widely
+applicable as possible, while trying to deliver examples of real world
+solutions.
+
+This paper includes a
+[benchmark](https://github.com/ome/bioimage-latency-benchmark) which measures
+the access times for different types of  data chunks. The reading performance
+of NGFFs is compared to the established TIFF and HDF5 format for different
+imaging modalities in different types of storage. These results demonstrate
+the benefits of the different formats under different access scenarios, e.g.,
+on a local computer, in the cloud, etc..  We discuss the advantages and
+tradeoffs of each format in various contexts.  We believe there is no single
+format that provides optimal performance in all scenarios. The corollary to
+this is that any imaging data-intensive project will have to consider these
+issues and make informed choices regarding the data structures they use. See
+the “[Outlook](https://www.nature.com/articles/s41592-021-01326-w#Sec4)”
+section of the paper for more details.
+
+## NGFF specifications
+
+In addition to the paper, several additions have been made to the
+[latest NGFF specification](https://ngff.openmicroscopy.org/latest/), currently at version 0.3.
+
+### Dimension separator
+
+Preliminary results for the latency benchmark of the NGFF paper revealed
+performance issues when accessing Zarr chunks remotely for some modalities.
+Our tests showed that the source of these issues was related to the separator
+used between chunks in the Zarr format. The version
+[0.2](https://ngff.openmicroscopy.org/0.2) includes a backwards incompatible
+change to use slash (`/`) as the separator between individual chunks rather
+than dot (`.`). This separator must be used for all multiscale images of all
+modalities, including label images and high-content screening fields of views.
+Moving forward, this restriction will be eased, but the underlying libraries
+like Zarr will be moving to `/` as the default.
+
+### 2D-5D axes
+
+Until version 0.2, the axes of multiscales images were implicitly assumed to
+be XYZCT. The [0.3](https://ngff.openmicroscopy.org/0.3/) version of the NGFF
+specification loosened this requirement by introducing a mandatory axes
+attribute in the `multiscales` specification. This extends  dimensionality
+between 2D and 5D. For instance, it is possible to store a time-lapse 2D image
+using XYT or a three-dimensional volume as XYZ.
+
+### Sample NGFF datasets
+
+The table below lists a comprehensive set of NGFF samples generated from IDR sources to cover all the features introduced since 0.1.
+
+<div class="row">
+    <div class="small-12 small-centered medium-12 medium-centered columns">
+        <div class="row horizontal">
+            <table>
+                <thead>
+                    <th>Image</th>
```

Comment:
Probably another CSS issue it's a bit unclear that the image thumbnails are clickable, but that may be just me.